bundle.Dockerfile

```diff
@@ -7,7 +7,7 @@ LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=ibm-namespace-scope-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=v4.0
 LABEL operators.operatorframework.io.bundle.channel.default.v1=v4.0
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.28.0+git
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.24.0
```

Comment:
please update your operator-sdk, we should avoid downgrading versions even if for the most part the operator-sdk version doesn't really impact functionality